references/detection/train_tpu.py

```diff
@@ -133,6 +133,7 @@ def do_prediction(image_path, use_cpu=False):
     model.eval()
 
     image = Image.open(image_path)
+    image = torchvision.transforms.functional.resize(image, (800, 600))
     image_tensor = torchvision.transforms.functional.to_tensor(image)
 
     if not use_cpu:
```

Comment:
Not needed for this PR, but we probably don't need `torch._XLAC._xla_set_default_device(...)` anymore as that's been updated.

Reply:
Hmm not sure what you're referring to here, I don't think we call that function anywhere?

Reply:
Ah sorry that was confusing. Check line 142.